src/redux/config.js

```diff
@@ -29,14 +30,20 @@ const isMacOS = navigator.userAgent.indexOf('Mac') !== -1
 
 const PUB_REST_API_URL = isElectronApp ? 'http://localhost:45001' : process.env.REACT_APP_UFX_PUBLIC_API_URL
 
+const RC_KEYWORD = '-rc'
+
 const appVersion = process.env.npm_package_version
 
+const isRCVersion = _includes(appVersion, RC_KEYWORD)
```

Comment:
is it possible to use same get_remote_version selector instead of this new one?

Reply:
no, remote version is a last product version